.github/workflows/build-test.yml

```diff
@@ -19,6 +19,9 @@ on:
       lib_write_deadline:
         type: string
         default: "OFF"
+      lib_force_host_verification:
+        type: string
+        default: "OFF"
```

Comment:
Again, I think default should be "ON". This is how the CMakeFile is, and how we want it to be. What we may want is a matrix run where we disable it, and then on that matrix run, we could even run only test that start with `SSL`...

Reply:
0/5 leave it as a separate job for now (I reversed the default/new job settings).

The last time I tried to use the matrix fully I ran into a couple issues, the biggest being that the jobs (names) were difficult to distinguish in the workflow view; I'll try again soon.